src/styles/blocks/main-wrapper.scss

```diff
@@ -0,0 +1,10 @@
+.main-wrapper {
+  display: flex;
+  flex-direction: column;
+
+  &__card {
+    max-width: 1024px;
+    padding: 50px 0 71px;
+    margin: 0 auto;
+  }
```

Comment:
don't combine internal and external styles for BEM block

Reply:
But in this place i use mix, and  internal and external styles for BEM element.

Reply:
To fix it add one more wrapper and put some styles there. 
Or you can leave like this and hope for luck, and mentors good mood😂😅

Reply:
It is okay, but better to move `paddings` to bem block card